packages/graphql_server/src/test/githubApi.test.ts

```diff
@@ -7,9 +8,34 @@ export async function testGetRepositoryTopics(repoFounder: string, repoName: str
 
 //test function that calls the method and prints out all contributors
 export async function testGetContributorCount(owner: string, repo: string) {
-  console.log(await getContributorCount(owner, repo, ' ')) //here we need to pass an auth token
+  console.log(await getContributorCount(owner, repo, '')) //here we need to pass an auth token
 }
 
+async function contributorCountForTrendingRepos() {
```

Comment:
refactor the for loops into a function which you can call  with 'daily', 'weekly' or 'monthly' as params.